src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -28,29 +37,31 @@
    * variable used to keep track of activity, I'm thinking to create more variables as we go to
    * register different types of activities
    */
-  private List<String> activity;
-  /** used to keep track of users friends */
-  private List<String> friends_usernames;
+
+  /** Store class that gives access to Users. */
+  private ActivityStore activityStore;
+
+  /** List of all activities */
+  private List<String> activities;
+
+
+
 
   @Override
   public void init() throws ServletException {
     super.init();
-
-    /** hardcoded activity for now */
-    activity = Arrays.asList("Ricardo Joined", "Elona says hi", "Drew just left the conversation");
-    friends_usernames = Arrays.asList("Ricardo", "Elona", "Drew");
+    setActivityStore(activityStore.getInstance());
   }
 
+  void setActivityStore(ActivityStore activityStore){
+    this.activityStore = activityStore;
+  }
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    String username = (String) request.getSession().getAttribute("user");
-
-    if (username == null) {
-      response.sendRedirect("/login");
-      return;
-    } else {
-      request.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp").forward(request, response);
+    List<Activity> activities = activityStore.getAllActivities();
+    request.setAttribute("activities", activities);
+    request.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp").forward(request, response);
     }
```

Comment:
This brace here is misaligned, it should be two spaces to the left

Reply:
done!